tests/template/t18113.nim

```diff
@@ -0,0 +1,14 @@
+# ensure template pragma handling doesn't eagerly attempt to add an implicit
+# 'pushed' pragma to the evaluation of any intermediate AST prior to
+# substitution.
+ 
+# bug #18113
+
+import sequtils
+
+{.push raises: [Defect].}
```

Comment:
`raises: [Defect]` doesn't make a lot of sense even if it was in #18113; instead this tests should use a push pragma that makes sense and reproduced the bug, eg:
```nim
{.push inline.}
var a = toSeq([1, 2, 3, 5, 10]).filterIt(it > 5)
{.pop.}
```